vendor_download.sh

```diff
@@ -136,6 +142,57 @@ echo "Lernie download successful"
 
 rm $LERNIE_RELEASE_JSON
 
+# Download latest release of Circe
+
+echo "Downloading circe binary from latest release"
+CIRCE_RELEASE_JSON=vendor-bins/circe-release.json
+curl -sSL \
+    -H "Authorization: token $GITHUB_TOKEN" \
+    -H "Accept: application/vnd.github.v3.raw" \
+    https://api.github.com/repos/fossas/circe/releases/latest > $CIRCE_RELEASE_JSON
+
+CIRCE_TAG=$(jq -cr ".tag_name" $CIRCE_RELEASE_JSON)
+echo "Using circe release: $CIRCE_TAG"
+
+if [[ "$OSTYPE" == "msys" || "$OSTYPE" == "cygwin" || "$OSTYPE" == "win32" ]]; then
+  # Windows uses .zip files
+  CIRCE_ARCHIVE_NAME="circe-$CIRCE_ASSET_POSTFIX.zip"
+  echo "Downloading Windows $CIRCE_ARCHIVE_NAME"
+  CIRCE_DOWNLOAD_URL="https://github.com/fossas/circe/releases/download/$CIRCE_TAG/$CIRCE_ARCHIVE_NAME"
```

Comment:
Why get the URL for Circe like this? Other bins like Lernie and Themis seem to extract it directly out of the releases JSON: https://github.com/fossas/fossa-cli/blob/73610a04f93b8f48cd4b1454f96fc5729de83bf5/vendor_download.sh#L132-L139

Reply:
good catch! I'll update.

Reply:
actually, this requires a bit larger of a change than I'm interested in doing at this point and it works as is- do you have an objection to this merging as is?

Reply:
No objections